doc/tutorials/using_partition_mode_on_up2.rst

```diff
@@ -0,0 +1,136 @@
+.. _partition_mode:
+
+Using partition mode on UP2
+===========================
+This tutorial describes all the steps you need to take in order to use partition mode with the ACRN hypervisor on UP2.
```

Comment:
Please keep line length < 80 characters when possible

Reply:
@dbkinder PR(https://github.com/projectacrn/acrn-hypervisor/pull/1558) has extended line length
to 120 instead of 80. Should we keep align with it.